### ISSUE TITLE
Preview Window does not display font color properly

### ISSUE THREAD

**Issue by Mertsch** [inbasic/ignotifier#371]
In the following picture you can see that in gmail (left) you can read the content, while in preview it's black on (almost) black

![gmail notifier preview window color bug](https://cloud.githubusercontent.com/assets/9402861/11687120/0d26d4f6-9e85-11e5-8b38-953e17ea30e3.png)

**Comment by inbasic** [comment]
What theme are you using?

**Comment by Mertsch** [comment]
@inbasic https://addons.mozilla.org/en-US/firefox/addon/mx4/

**Comment by inbasic** [comment]
I am unable to install this theme on Mac OS. Have you had this issue with another theme as well?

**Comment by Mertsch** [comment]
@inbasic I have switched back to the default Fx 44.0 theme on Windows 10 and the same affect happens.
Also it happens with Win 7 and MX4 (I did not try default theme there)

**Comment by Mertsch** [comment]
@inbasic A bit more research ...
The color used in the preview Window is the one defined within the Fx settings under Content --> Colors
![ColorSetting](https://cloud.githubusercontent.com/assets/9402861/12290430/0e24267e-b9e2-11e5-9cb3-bc906953453c.png)

**Comment by inbasic** [comment]
I see, can you try this dev version and let me know if the issue is fixed or not?
https://github.com/inbasic/ignotifier/blob/beta/builds/packed/firefox.xpi?raw=true

**Comment by Mertsch** [comment]
I did not work.
The "Colors" setting is now ignored, but the outcome is still the same (black font color)

**Comment by Mertsch** [comment]
@inbasic Looking at the source of the email, it seems inline styles are ignored
Here is the original source
```
style="padding: 20px; font-size: 12px; line-height: 17px; color: rgb(198, 212, 223); font-family: Arial,Helvetica,sans-serif;"
```

**Comment by inbasic** [comment]
Can you send me a sample email with this problem? inb[dot]cor[at]gmail[dot]com

**Comment by Mertsch** [comment]
@inbasic You have mail

**Comment by inbasic** [comment]
@Mertsch I haven't got your email.

**Comment by Mertsch** [comment]
@inbasic I sent you another mail, did you get that?
You can also try to contact me at my user name at gmail.com

**Comment by Mertsch** [comment]
@inbasic Sent you the mail :)

**Comment by inbasic** [comment]
@Mertsch Thanks for the email forwarding. The issue is fixed now
https://github.com/inbasic/ignotifier/blob/beta/builds/packed/firefox.xpi?raw=true

btw, I have deleted your email per your request.

**Comment by Mertsch** [comment]
Thank you for your time and effort.
This addon replaced my desktop email notifier app completely. Keep up the good work

**Comment by Mertsch** [comment]
Fix Confirmed with 0.6.2
